guppylang/module.py

```diff
@@ -113,28 +114,17 @@ def load(
                 names[alias or imp.name] = imp.id
                 modules.add(module)
             elif isinstance(imp, GuppyModule):
-                # TODO: In the future this should be a qualified module import. For now
-                #  we just import all contained definitions that don't start with `_`
                 imp.check()
-                imports.extend(
-                    (defn.name, defn)
-                    for defn in imp._globals.defs.values()
-                    if not defn.name.startswith("_")
-                )
+                def_id = DefId.fresh(imp)
+                name = alias or imp.name
+                defn = ModuleDef(def_id, name, None, imp._globals)
```

Comment:
Would be nice, but we don't have an AST for module definitions since they are defined by executing regular Python code instead of us parsing something